### ISSUE TITLE
Import all Markdown files in a directory

### ISSUE THREAD

**Issue by davidak** [codimd/cli#32]
This should work:

`codimd import /path/to/markdown/files`

## Workaround

```
[davidak@X230:~/code/cli/bin]$ for file in ~/Downloads/hackmd/*.md; do xdg-open $(CODIMD_SERVER='https://pad.nixnet.services' ./codimd import "$file" | awk '{print $4}'); done
```
(You have to open the imported note for it to appear in the history.)

**Comment by pirate** [comment]
I don't think this is worth implementing tbh, the bash loop is simple enough.

```bash
export CODIMD_SERVER='https://pad.nixnet.services'

for file in ~/Downloads/hackmd/*.md
	./codimd import "$file" >> note_urls.txt
done
```

The URL output will be fixed by #31.

**Comment by deletosh** [comment]
If a search brought you here, here's a post-codimd version: 

```sh
export HEDGEDOC_SERVER='https://notes.example.com'  

for file in ~/Downloads/hackmd/*.md 
do
	hedgedoc import "$file" >> note_urls.txt
done
```
